lxd/certificates.go

```diff
@@ -354,6 +354,50 @@ func clusterMemberJoinTokenValid(d *Daemon, r *http.Request, projectName string,
 	return nil, nil
 }
 
+// certificateTokenValid searches for certificate token that matches the add token provided.
+// Returns matching operation if found and cancels the operation, otherwise returns nil.
+func certificateTokenValid(d *Daemon, r *http.Request, projectName string, addToken *api.CertificateAddToken) (*api.Operation, error) {
+	ops, err := operationsGetByType(d, r, projectName, db.OperationCertificateAddToken)
```

Comment:
`projectName` shouldn't really be relevant here as certificates are considered a global resource so will always be in default